src/main/java/uniresolver/driver/did/abt/DidAbtDriver.java

```diff
@@ -0,0 +1,137 @@
+package uniresolver.driver.did.abt;
+
+import did.Authentication;
+import did.DIDDocument;
+import did.PublicKey;
+import did.Service;
+import org.apache.http.HttpEntity;
+import org.apache.http.client.HttpClient;
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.impl.client.HttpClientBuilder;
+import org.apache.http.impl.client.LaxRedirectStrategy;
+import org.apache.http.util.EntityUtils;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.json.JSONObject;
+import uniresolver.ResolutionException;
+import uniresolver.driver.Driver;
+import uniresolver.result.ResolveResult;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class DidAbtDriver implements Driver {
+
+	private static Logger log = LoggerFactory.getLogger(DidAbtDriver.class);
+
+	public static final Pattern DID_ABT_PATTERN = Pattern
+			.compile("^did:abt:([123456789ABCDEFGHJKLMNPQRSTUVWXYZabcdefghijkmnopqrstuvwxyz]{35,36})$");
```

Comment:
this will limit what's possible in the DID.  I want it just be anything.

Reply:
Fixed